src/styles/bloks/stars.css

```diff
@@ -0,0 +1,24 @@
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  height: 16px;
+  width: 16px;
```

Comment:
The height and width of the stars are set to 16px here, whereas in the main CSS file, they are set to 14px. Ensure that the size is consistent across all stylesheets to maintain a uniform appearance.